test/api.js

```diff
@@ -565,66 +554,6 @@ function generateTests(prefix, apiCreator) {
 		return api.run([path.join(__dirname, 'fixture/immediate-3-exit.js')]);
 	});
 
-	test(prefix + 'testing nonexistent files causes an AvaError to be emitted', function (t) {
-		t.plan(2);
-
-		var api = apiCreator();
-
-		api.on('test-run', function (runStatus) {
-			runStatus.on('error', function (err) {
-				t.is(err.name, 'AvaError');
-				t.match(err.message, /Couldn't find any files to test/);
-			});
-		});
-
-		return api.run([path.join(__dirname, 'fixture/broken.js')]);
-	});
-
-	test(prefix + 'test file in node_modules is ignored', function (t) {
-		t.plan(2);
-
-		var api = apiCreator();
-
-		api.on('test-run', function (runStatus) {
-			runStatus.on('error', function (err) {
-				t.is(err.name, 'AvaError');
-				t.match(err.message, /Couldn't find any files to test/);
-			});
-		});
-
-		return api.run([path.join(__dirname, 'fixture/ignored-dirs/node_modules/test.js')]);
-	});
-
-	test(prefix + 'test file in fixtures is ignored', function (t) {
-		t.plan(2);
-
-		var api = apiCreator();
-
-		api.on('test-run', function (runStatus) {
-			runStatus.on('error', function (err) {
-				t.is(err.name, 'AvaError');
-				t.match(err.message, /Couldn't find any files to test/);
-			});
-		});
-
-		return api.run([path.join(__dirname, 'fixture/ignored-dirs/fixtures/test.js')]);
-	});
-
-	test(prefix + 'test file in helpers is ignored', function (t) {
```

Comment:
Those tests are not deleted, they moved to `ava-files.js` tests, which covers these cases.